### PR TITLE
docs: Add Microsoft Azure installation instructions

### PR DESCRIPTION
**Pull request by ahmetb**
Adding instructions that refers to the Microsoft Azure website on creating Docker-ready Linux VMs on the platform.

Signed-off-by: Ahmet Alp Balkan ahmetalpbalkan@gmail.com
cc: @SvenDowideit @squillace

### CODE REVIEW

**Comment by moxiegirl** [comment]
@ahmetalpbalkan Is there a reason we don't want to put the actual instructions on the Docker site?

**Comment by jfrazelle** [comment]
@moxiegirl We don't really have instructions per cloud provider because you
_can_ run docker on any cloud provider

**Comment by jfrazelle** [comment]
Actually nm I guess we do I had never seen those Rackspace ones

**Comment by ahmetalpbalkan** [comment]
@moxiegirl if you click on them they are really long than any other similar doc we have here. Also if we duplicate the content, that will be a maintenance cost. Also this small doc refers to 3 different ways to do the same thing. Will be a very long and unreadable one if we bring all that content here.

**Comment by moxiegirl** [comment]
@ahmetalpbalkan If we refer them to installation docs that we don't write, we imply that we approve or have tested the content just as we would have had we written them ourselves. We are trying to simplify both the maintenance and the support of our installation documents. So, I'd rather just pick one way Docker recommends and go with that. That way we can test *our instructions* if other people want to document in detail other ways to do the same thing...god bless them.

Do you find one blog provides a better instruction than another?  

BTW, I'm not suggesting we hold up this PR waiting for a Dockerized installation set. I can open another PR for that.

**Comment by squillace** [comment]
@moxiegirl @ahmetalpbalkan Hi there -- I'm curious: As Ahmet points out, you actually have one doc for **each** cloud provider about how to install there, and each one points to THEIR documentation -- they do NOT recreate their steps inside your docs. Two other quick examples are:

- https://github.com/docker/docker/blob/master/docs/sources/installation/amazon.md
- https://github.com/docker/docker/blob/master/docs/sources/installation/google.md

they all pretty much do the same thing as Ahmet's PR. They say, hey, here's what you do to get going (but the what you do part points to their documentation, not yours). Since there's about five ways to get docker running on Azure in different configurations to support different scenarios, I'm curious why you would want all that on your site, given that you already do what Ahmet has submitted so far as I can see.... 

I guess I'm not clear on your phrase, "I'd rather just pick one way Docker recommends and go with that." -- given that we thought we **were** doing what the others were doing. :-) -- Cheers, Ralph

**Comment by moxiegirl** [comment]
@squillace We are looking at simplifying the installation section on our website. That is where my statement is coming from. Those changes are coming. Obviously, I jumped the gun in bringing that topic into this discussion, my apologies for that. 

LGTM

**Comment by squillace** [comment]
@moxiegirl Ah, I completely understand. Of course, we're figuring out the same issue in the reverse direction. 

FWIW, we generally think that basic installation on an OS should be "all you", and then each cloud or platform or orchestration api might then have automation or custom installation commands or things that they should document. We would be happy to write the basic steps for those on your site as well, but as you should not expect to support every platform's optimizations for docker/machine/swarm/other stuff, it's always good to have the ability to have a "link-out" signal (Wikipedia does this for example), which tells the reader "these are the official instructions over THERE, but **complaints** about them go over there, too." That kind of thing. (For example, we always point to your HTTPS docs for ensuring or modifying secure connections -- we don't expect that we would ever know better than you, unless you're using **our** commands/cli....)

This is just for your consideration -- YMMV, and we'll follow your lead here. 

 Feel free to ping me or @ahmetalpbalkan when you have a forward plan. Cheers!

**Comment by moxiegirl** [comment]
@squillace Cool...sounds like we are swimming in the same ocean.  Meaning, I think we are on the same page re install docs.  More later...thanks for your understanding.

**Comment by ahmetalpbalkan** [comment]
rebased due to merge conflicts.

File: docs/sources/installation/azure.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+page_title: Installation on Microsoft Azure Platform
+page_description: Instructions for creating a Docker-ready virtual machine on Microsoft Azure cloud platform.
+page_keywords: Docker, Docker documentation, installation, azure, microsoft
+
+# Microsoft Azure
+
+
+## Creating a Docker host machine on Azure
+
+Please check out to the following detailed tutorials on [Microsoft Azure][0] website to find out different ways to create a Docker-ready Linux virtual machines on Azure:
```
Comment (SvenDowideit) [inline docs/sources/installation/azure.md]: 80 char line wraps please (GH diffs are terrible on long lines)

Comment (ahmetalpbalkan) [inline docs/sources/installation/azure.md]: @SvenDowideit ah, my markdown editor has failed me on this. done.

**Comment by SvenDowideit** [comment]
LGTM

**Pull request merged**